dashboard/test/controllers/api/v1/assessments_controller_test.rb

```diff
@@ -39,22 +50,188 @@ class Api::V1::AssessmentsControllerTest < ActionController::TestCase
     assert_equal '{}', @response.body
   end
 
+  test "verified teacher should get assessments structure" do
+    # Sign in and create a new script.
+    sign_in @teacher
+    script = create :script
+
+    # Set up an assessment for that script.
+    sub_level1 = create :text_match, name: 'level_free_response', type: 'TextMatch'
+    sub_level2 = create :multi, name: 'level_multi_unsubmitted', type: 'Multi'
+    sub_level3 = create :multi, name: 'level_multi_correct', type: 'Multi'
+    sub_level4 = create :multi, name: 'level_multi_incorrect', type: 'Multi'
+    sub_level5 = create :multi, name: 'level_multi_unattempted', type: 'Multi'
+
+    level1 = create :level_group, name: 'LevelGroupLevel1', type: 'LevelGroup'
+    level1.properties['title'] =  'Long assessment 1'
+    level1.properties['pages'] = [{levels: ['level_free_response', 'level_multi_unsubmitted']}, {levels: ['level_multi_correct', 'level_multi_incorrect']}, {levels: ['level_multi_unattempted']}]
```

Comment:
nit: break this into multiple lines (same for `expected_answers` and `expected_questions` below)